### DISCUSSION TITLE
Why does authentik make requests on the internet?

### DISCUSSION THREAD

**Discussion by Paulpatou** [goauthentik/authentik#5609] [Q&A] [answered]
Hello,
First of all, thank you for your work on this great tool

I am testing Authentik in an environment that does not access the internet and in the logs I keep getting errors like this:

`authentik-worker-1      | {"event": "Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7fa08aa050>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /api/4504163677503489/envelope/", "level": "warning", "logger": "urllib3.connectionpool", "timestamp": 1684149975.0229588}`

What is the reason for this?
Can this generate slowdowns on page loads? If yes, what can be done to improve it?

**Comment by BeryJu** [comment, marked as answer]
See https://goauthentik.io/docs/installation/air-gapped for which URLs are requested and how to disable the corresponding features

**Reply by Paulpatou** [reply]
Thanks for the quick answer, I hadn't found it in the documentation.
